src/messages.rs

```diff
@@ -267,6 +263,9 @@ impl RoutingMessage {
 #[derive(PartialEq, Eq, PartialOrd, Ord, Clone, Debug, RustcEncodable, RustcDecodable)]
 pub struct SignedMessage {
     encoded_body : Vec<u8>,
+    claimant     : Address,
```

Comment:
Oh, sorry, missed that the type is `Address`. It will be redundant in both cases.